### PR TITLE
Support to link es6.x and run gor with docker

### PR DESCRIPTION
**Pull request by ShaoNianyr**

### CODE REVIEW

File: elasticsearch.go [hunk 1]
```diff
@@ -1,171 +1,92 @@
...
 	"log"
-	"net/url"
-	"strings"
-	//"regexp"
 	"time"
-
 	"github.com/buger/goreplay/proto"
-
-	"github.com/mattbaird/elastigo/lib"
+	"github.com/olivere/elastic"
```
[the remark this answers is not in the document]
Comment (ShaoNianyr) [reply elasticsearch.go:13]: update

[the remark this answers is not in the document]
Comment (ShaoNianyr) [reply elasticsearch.go:13]: updated

**Comment by ShaoNianyr** [comment]
conflicts resolved

**Comment by rtkgjacobs** [comment]
Any chance we can get these new conflicts resolved?  Bumping this PR as it seems to have gone stale.  I'm interested in this feature working also. 

Pinging @slimus , @ShaoNianyr  and @urbanishimwe  - thanks. 

See related issue https://github.com/buger/goreplay/issues/421#issuecomment-811192356

**Comment by ShaoNianyr** [comment]
It hasn't merged for long time that there's a lot of conflicts. I will initiate a new branch and make sure it works.

**Pull request closed without merging**